### PR TITLE
Add status icon to toobar buttons.

### PR DESCRIPTION
**Pull request by themithy**

### CODE REVIEW

File: packages/neptune-extension/src/labextension/components/ToolbarButton.less [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-    padding-left: 23px !important;
-  }
+.neptune-icon {
+  background-image: url('../../common/assets/logo.svg') !important;
+  background-size: 15px !important;
+  background-position: 2px center !important;
+  background-repeat: no-repeat !important;
 }
+
+.status-icon-green {
```
Comment (jakubsy) [inline packages/neptune-extension/src/labextension/components/ToolbarButton.less]: IMHO it is wrong, that you add style to icons to component file, and they are defined outside this component.

File: packages/neptune-extension/src/nbextension/components/ToolbarButton.less [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
-  &--icon {
-    background-image: url('../../common/assets/logo.svg') !important;
-    background-size: 15px !important;
-    background-position: 8px center !important;
-    background-repeat: no-repeat !important;
-    padding-left: 25px !important;
+.neptune-icon {
+  background-image: url('../../common/assets/logo.svg') !important;
+  background-size: 15px !important;
+  background-position: 2px 4px !important;
```
Comment (jakubsy) [inline packages/neptune-extension/src/nbextension/components/ToolbarButton.less]: in vertical aligment I suggest using center

File: packages/neptune-extension/src/labextension/components/ToolbarButton.less [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-  min-width: 27px !important;
 
-  &--logo {
-    background-image: url('../../common/assets/logo.svg') !important;
-    background-size: 15px !important;
-    background-position: 6px center !important;
-    background-repeat: no-repeat !important;
-    padding-left: 23px !important;
-  }
+.neptune-icon {
```
Comment (jakubsy) [inline packages/neptune-extension/src/labextension/components/ToolbarButton.less]: IMHO move it to common styles

File: packages/neptune-extension/src/labextension/components/ToolbarButton.less [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
+  background-image: url('../../common/assets/logo.svg') !important;
+  background-size: 15px !important;
+  background-position: 2px center !important;
+  background-repeat: no-repeat !important;
 }
+
+.status-icon-green {
+  color: green;
```
Comment (jakubsy) [inline packages/neptune-extension/src/labextension/components/ToolbarButton.less]: Please consult color with Ola

File: packages/neptune-extension/src/nbextension/components/ToolbarButton.less [hunk 5]
```diff
@@ -1,20 +1,22 @@
...
+  background-repeat: no-repeat !important;
+
+  height: 14px;
+  width: 20px;
+}
+
+.status-icon-green {
```
Comment (tawez) [inline packages/neptune-extension/src/nbextension/components/ToolbarButton.less]: What if we change the color to blue?
Should we change the class name too?

**Pull request closed without merging**